doc/api/dns.md

```diff
@@ -15,8 +15,8 @@ For example, looking up `iana.org`.
 ```js
 const dns = require('dns');
 
-dns.lookup('iana.org', (err, addresses, family) => {
-  console.log('addresses:', addresses);
+dns.lookup('iana.org', (err, address, family) => {
+  console.log('address:', address);
```

Comment:
It would likely be worthwhile showing two examples, this one and one with `all = true` so that `address` becomes an Array.